### ISSUE TITLE
Wrong member order in ObjectInfo type

### ISSUE THREAD

**Issue by jseldent** [eProsima/Micro-XRCE-DDS#137]
Micro-XRCE-DDS reverses the order of the `activity` and  `config` members in `ObjectInfo`. According to the official [dds_xrce_types.idl](https://www.omg.org/spec/DDS-XRCE/20190301/dds_xrce_types.idl), it should be
``` idl
    @extensibility(FINAL)
    struct ObjectInfo {
        @optional  ActivityInfoVariant  activity;
        @optional  ObjectVariant        config;
    };
```
But an actual INFO message looks like
![screenshot](https://user-images.githubusercontent.com/57678901/160435791-db776f1c-08c1-4a83-803b-f8b362d31c61.png)
which has `config` first.

**Comment by pablogs9** [comment]
Is the dissector also in the wrong order?

**Comment by jseldent** [comment]
Yes. That's how I discovered the bug, because I couldn't properly dissect the INFO message if I didn't swap the order.

**Comment by pablogs9** [comment]
We will take this into account in next release
